src/style.css

```diff
@@ -0,0 +1,119 @@
+body {
+  margin: 0;
+  padding: 0;
+  background: #e5e5e5;
+  font-family: "Roboto", sans-serif;
+
+  --button-color: #00acdc;
+}
+
+a {
+  text-decoration: none;
+}
+
+.product {
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/217772891-e5764df3-7bc2-48c4-9d7a-c696488f2c23.png)
Move these styles to a separate css file